src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -28,11 +34,18 @@ export const AttachmentLinkUnStyled = ({ className, filename, data, type }: Atta
         <div className="col-12 col-md">
           <p className="filetext">
             <span className="filename">{filename}</span>
-            <span className="filesize">({filesize})</span>
+            {hasBase64 && <span className="filesize">({filesize})</span>}
           </p>
-          <a href={`data:${type};base64,${data}`} download={`${filename}`}>
-            Download
-          </a>
+          {hasBase64 && (
+            <a href={`data:${type};base64,${data}`} download={`${filename}`}>
+              Download
+            </a>
+          )}
+          {path && (
+            <a href={path} download={`${filename}`}>
+              Download
+            </a>
+          )}
```

Comment:
Was thinking if it will be more meaningful to have the entire block clickable instead of just the `Download` text

Reply:
okay np